components/quizzes/numberedcircles.tsx

```diff
@@ -3,9 +3,8 @@ import React from "react";
 const NumberedCircles = () => {
   return (
     <div className="relative grid grid-cols-1 gap-y-8 justify-center items-center">
-      {" "}
       <div className="h-12 w-0.5 bg-gray-400 absolute top-12 left-6"></div>
-      <div className="w-12 h-12 rounded-full bg-brandPrimary flex items-center justify-center text-white font-bold mb-2">
```

Comment:
I prefer to use the themed colors, so we could use this quiz logic for other brands.

Reply:
right, was having trouble accessing any of the primary charmander colour because of an error in Layout.tsx I think because they are non-logged in users. I